lib/super_good/solidus_taxjar/api.rb

```diff
@@ -10,6 +10,9 @@ def self.default_taxjar_client
 
       def initialize(taxjar_client: self.class.default_taxjar_client)
         @taxjar_client = taxjar_client
+        @taxjar_client.set_api_config('headers', {
+          'x-api-version' => '2020-08-07'
+        })
```

Comment:
Would it make more sense to inject this behaviour into `.default_taxjar_client`? This dependency is only injected for testing purposes; I don't expect anyone is injecting their own client.

Reply:
Good point! Updated!